realis.go

```diff
@@ -563,29 +554,23 @@ func (r *realisClient) GetInstanceIds(key *aurora.JobKey, states map[aurora.Sche
 func (r *realisClient) GetJobUpdateSummaries(jobUpdateQuery *aurora.JobUpdateQuery) (*aurora.Response, error) {
 	var resp *aurora.Response
 	var err error
-
 	defaultBackoff := r.config.backoff
-	duration := defaultBackoff.Duration
-	for i := 0; i < defaultBackoff.Steps; i++ {
-		if i != 0 {
-			adjusted := duration
-			if defaultBackoff.Jitter > 0.0 {
-				adjusted = Jitter(duration, defaultBackoff.Jitter)
+	xerr := ExponentialBackoff(*defaultBackoff, func() (bool, error) {
+		resp, err = r.readonlyClient.GetJobUpdateSummaries(jobUpdateQuery)
+		if err != nil {
```

Comment:
Leave a TODO here for differentiating between retryable errors and non-retryable errors